server/src/test/java/org/elasticsearch/ExceptionSerializationTests.java

```diff
@@ -133,11 +133,7 @@ public void testExceptionRegistration()
 
             @Override
             public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) throws IOException {
-                Path next = pkgPrefix.resolve(dir.getFileName());
-                if (ignore.contains(next)) {
-                    return FileVisitResult.SKIP_SUBTREE;
-                }
-                pkgPrefix = next;
```

Comment:
Are you sure it's ok to remove this line ? Line 189 reads it.

Reply:
My reasoning is this: nothing in this method in this current form changes "next" (the contains() call should have no side effects), so next is still "pkgPrefix.resolve(dir.getFileName())" here, which is what I use to set pkgPrefix in the following line. Does that make sense? I'm not fully sure what the ignore.contains() check was supposed to do at some point, but I think it doesn't do anything now.